src/mediaplayer.cs

```diff
@@ -261,15 +269,26 @@ interface MPMediaItemArtwork {
 		[Export ("imageWithSize:")]
 		[return: NullAllowed]
 		UIImage ImageWithSize (CGSize size);
+#else
+		[Export ("initWithBoundsSize:requestHandler:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGSize boundsSize, Func<CGSize, NSImage> requestHandler);
+
+		[Export ("imageWithSize:")]
+		[return: NullAllowed]
+		NSImage ImageWithSize (CGSize size);
```

Comment:
no verb/action in the method name, better to use something like `NSImage  GetImage (CGSize size);`